language-plutus-core/src/Language/PlutusCore/Evaluation/Machine/Ck.hs

```diff
@@ -188,13 +184,13 @@ applyEvaluateCkBuiltinName name args = do
 -- if @F@ does not compute, so we simply do not introduce a rule that can't possibly fire.
 evaluateCk
     :: (GShow uni, GEq uni, DefaultUni <: uni, Closed uni, uni `Everywhere` ExMemoryUsage)
-    => Term TyName Name uni () -> Either (CkEvaluationException uni) (Term TyName Name uni ())
-evaluateCk term = [] |> term
+    => Plain Term uni -> Either (CkEvaluationException uni) (WithMemory Term uni)
+evaluateCk term = [] |> withMemory term
 
 -- | Evaluate a term using the CK machine. May throw a 'CkMachineException'.
 unsafeEvaluateCk
     :: ( GShow uni, GEq uni, DefaultUni <: uni, Closed uni, uni `Everywhere` ExMemoryUsage
        , Typeable uni, uni `Everywhere` PrettyConst
        )
-    => Term TyName Name uni () -> EvaluationResultDef uni
+    => Plain Term uni -> EvaluationResult (Term TyName Name uni ExMemory)
```

Comment:
Since `ExMemory` stuff is now all over the place, we have it also here. There are two options:

1. remove `ExMemory` from what the CEK machine returns (like it was before)
2. give up on having the same interface for both the CK and CEK machines and remove everything that relies on API being the same (only `Emit.hs`, I think?)

I'm going to try out 1 and if that doesn't work, then do 2 (perhaps in a different PR and perhaps regardless of whether 1 works out or not).